config/mainnet.go

```diff
@@ -156,10 +156,12 @@ func MainnetConfig() Config {
 			BeaconSyncWeightUnits:    800,
 		},
 		POET: activation.PoetConfig{
-			PhaseShift:        240 * time.Hour,
-			CycleGap:          12 * time.Hour,
-			GracePeriod:       1 * time.Hour,
-			RequestTimeout:    1100 * time.Second, // RequestRetryDelay * 2 * MaxRequestRetries*(MaxRequestRetries+1)/2
+			PhaseShift:                     240 * time.Hour,
+			CycleGap:                       12 * time.Hour,
+			GracePeriod:                    1 * time.Hour,
+			PositioningATXSelectionTimeout: 50 * time.Minute,
+			// RequestRetryDelay * 2 * MaxRequestRetries*(MaxRequestRetries+1)/2
```

Comment:
NIT: to make it clear what the comment explains:

```suggestion
			// RequestTimeout = RequestRetryDelay * 2 * MaxRequestRetries*(MaxRequestRetries+1)/2
```